src/jit/morph.cpp

```diff
@@ -14057,6 +14057,12 @@ GenTree* Compiler::fgMorphModToSubMulDiv(GenTreeOp* tree)
         denominator = fgMakeMultiUse(&tree->gtOp2);
     }
 
+    if (lvaLocalVarRefCounted)
```

Comment:
> Another odd thing is that we never seem to use numerator and denominator in the case where they are non-leaf-nodes, since their original use has been replaced by the comma with the temp assignment on the lhs, and the returned value is (as yet) unused

The returned values are used on line 14066 and 14068.